opaque_keys/edx/tests/test_asset_locators.py

```diff
@@ -91,12 +90,8 @@ def test_replace(self):
 
     def test_empty_path(self):
         with self.assertRaises(InvalidKeyError):
-            CourseKey.from_string('course-v1:org+course+run').make_asset_key('asset', '')
-
-        self.assertEquals(
-            '/c4x/org/course/asset/',
-            unicode(CourseKey.from_string('org/course/run').make_asset_key('asset', ''))
-        )
+            CourseKey.from_string('course-locator:org+course+run').make_asset_key('asset', '')
+            CourseKey.from_string('org/course/run').make_asset_key('asset', '')
```

Comment:
You should call `assertRaises` around each of these calls, rather than having only one assertion. As it stands, the second line (94) will never be tested.